VC/VehicleCore.rdf

```diff
@@ -3194,7 +3194,7 @@ Typical unit code(s): SEC for seconds&lt;br/&gt;&lt;br/&gt;
 	
 	<owl:ObjectProperty rdf:about="&auto-vc;bodyStyle">
 		<rdfs:subPropertyOf rdf:resource="&owl;topObjectProperty"/>
-		<rdfs:label xml:lang="en">body style</rdfs:label>
+		<rdfs:comment xml:lang="en">body style</rdfs:comment>
```

Comment:
https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md does not recommend using rdfs:comment - could you use some other property, e.g., the one for synonyms?